packages/lodestar/src/chain/factory/block/body.ts

```diff
@@ -25,6 +25,18 @@ export async function assembleBody(
   blockSlot: Slot,
   syncAggregateData: {parentSlot: Slot; parentBlockRoot: Root}
 ): Promise<allForks.BeaconBlockBody> {
+  // TODO:
```

Comment:
I wonder if we should, if number of agg attestations is < MAX_ATTESTATIONS, take some attestations from attestations memory pool. Assuming we delete attestation that are aggregated?

Reply:
Right, the logic to do that successfully is tricky. Since we where not doing that previously it should be done in another PR if we decide to.